packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -497,53 +501,54 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
     return _valueProperties;
   }
 
-  Map<String, Object> get valuePropertiesJson => json['valueProperties'];
+  Map<String, Object>? get valuePropertiesJson =>
+      json['valueProperties'] as Map<String, Object>?;
 
-  bool get hasChildren {
+  bool? get hasChildren {
     // In the summary tree, json['hasChildren']==true when the node has details
     // tree children so we need to first check whether the list of children for
     // the node in the tree was specified. If there is an empty list of children
     // that indicates the node should have no children in the tree while if the
     // 'children' property is not specified it means we do not know whether
     // there is a list of children and need to query the server to find out.
-    final List children = json['children'];
+    final List? children = json['children'] as List<dynamic>?;
     if (children != null) {
       return children.isNotEmpty;
     }
     return getBooleanMember('hasChildren', false);
   }
 
-  bool get isCreatedByLocalProject {
+  bool? get isCreatedByLocalProject {
     return getBooleanMember('createdByLocalProject', false);
   }
 
   /// Whether this node is being displayed as a full tree or a filtered tree.
-  bool get isSummaryTree => getBooleanMember('summaryTree', false);
+  bool? get isSummaryTree => getBooleanMember('summaryTree', false);
 
   /// Whether this node is being displayed as a full tree or a filtered tree.
-  bool get isStateful => getBooleanMember('stateful', false);
+  bool? get isStateful => getBooleanMember('stateful', false);
 
-  String get widgetRuntimeType => getStringMember('widgetRuntimeType');
+  String? get widgetRuntimeType => getStringMember('widgetRuntimeType');
 
   /// Check whether children are already available.
   bool get childrenReady {
-    return json.containsKey('children') || _children != null || !hasChildren;
+    return json.containsKey('children') || _children != null || !hasChildren!;
   }
 
-  Future<List<RemoteDiagnosticsNode>> get children async {
+  Future<List<RemoteDiagnosticsNode?>?> get children async {
```

Comment:
RemoteDiagnosticsNode does not need to be null

Reply:
done